pkg/verify_archive.bzl

```diff
@@ -122,7 +122,7 @@ def verify_archive_test(name, target,
         data = [target],
         python_version = "PY3",
         deps = [
-            "//pkg:verify_archive_test_lib",
+            Label("//pkg:verify_archive_test_lib"),
```

Comment:
This doesn't seem right. We shouldn't have to use Label() around deps like this.
What was your setup when you noticed the problem in the first place?
Were you using the distribution or trying to vendor in the code?

Reply:
I initially noticed the issue after bringing in `rules_pkg` as a dependency in my own project and creating a `verify_archive_test` target there. The error message made it look like it was searching for `//pkg:verify_archive_test_lib` from my own workspace instead of `rules_pkg`.

Reply:
Oh... I see it now. Because it's not a `py_test`, it's a *generated* `py_test`.